src/session.rs

```diff
@@ -57,6 +59,8 @@ pub(crate) struct Session {
     twcc_rx_register: TwccRecvRegister,
     twcc_tx_register: TwccSendRegister,
 
+    bwe: Option<SendSideBandwithEstimator>,
```

Comment:
Could also have

```rust
trait Bwe {}
enum BweImpl {
  SendSide(SendSideBandwithEstimator),
  NullBwe,
}
```

Instead of this optional business.

Reply:
Is "SendSide" a good name here? I understand this is as opposed to REMB.

@pthatcher thoughts?

Reply:
I saw this phrasing in libWebRTC, but I was thinking we should rename this struct anyway. I think it ought to be called `GoogccBwe` or something to that effect. In theory we could have other BWE strategies in the future which is an argument for the trait/enum I suggested above